### PR TITLE
Admin page mvp

### PR DESCRIPTION
**Pull request by lucieleblanc**
(finally)

### CODE REVIEW

**Comment by VillanuevaK** [comment]
Good work. I like the todo comment where you say you'll give users an 'isAdmin' property, but have that quick fix in the meanwhile.
There's also a conflict in the UserStore file that was probably caused by both of us working on the file, but it should be an easy fix.

File: src/main/java/codeu/controller/AdminServlet.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  private ConversationStore conversationStore;
+
+  /** Store class that gives access to Messages. */
+  private MessageStore messageStore;
+
+  /** Store class that gives access to Users. */
+  private UserStore userStore;
+
+  /** Store list of approved admins.
+   * TODO: give users an isAdmin property and determine access that way instead.
```
Comment (yufangqq) [inline src/main/java/codeu/controller/AdminServlet.java:33]: nit:
At google, normally TODO is followed by the ldap of the engineer or the bug number.
E.g.,
TODO(lucieleblanc)

File: src/main/java/codeu/model/store/basic/UserStore.java [hunk 2]
```diff
@@ -126,5 +126,30 @@ public boolean isUserRegistered(String username) {
...
+    User oldest = users.get(0);
+    User newest = users.get(0);
+    for (User user : users) {
+      if (user.getCreationTime().compareTo(oldest.getCreationTime()) < 0) {
+        oldest = user;
+      } else if (user.getCreationTime().compareTo(newest.getCreationTime()) > 0) {
+        newest = user;
+      }
+    }
+    List<String> topUsers = new ArrayList<>();
```
Comment (yufangqq) [inline src/main/java/codeu/model/store/basic/UserStore.java:175]: nit:
topUsers is not the best name for this variable.
Consider:
1) Rename it o oldestAndNewestUsers.
2) Or better, just create to String variable one called oldestUser, the other called newestUser.

Comment (yufangqq) [reply src/main/java/codeu/model/store/basic/UserStore.java:175]: Or maybe call it edgeUsers, or topAndBottomUsers? But again, I think keep two separate variables would be more readable.

File: src/test/java/codeu/controller/AdminServletTest.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        mockRequest = Mockito.mock(HttpServletRequest.class);
+        mockResponse = Mockito.mock(HttpServletResponse.class);
+        mockRequestDispatcher = Mockito.mock(RequestDispatcher.class);
+        Mockito.when(mockRequest.getRequestDispatcher("/WEB-INF/view/login.jsp"))
+                .thenReturn(mockRequestDispatcher);
+    }
+
+    /*
+
+    TODO: write POST test for allowed admins
```
Comment (yufangqq) [inline src/test/java/codeu/controller/AdminServletTest.java:38]: Add your username to the TODOs.

**Review by yufangqq — APPROVED** [review]

**Review by VillanuevaK — APPROVED** [review]

**Pull request merged**